index.json

```diff
@@ -24,6 +24,16 @@
       "active_record"
     ]
   },
+  "activestorage": {
+    "dependencies": [
+      "rails"
+    ],
+    "requires": [
+      "rails",
```

Comment:
It should not. Active Storage doesn't depend on Rails.

Reply:
Since ActiveStorage::Blob/ActiveStorage::Attachment are technically models, it needs to load the ActiveStorage engine, and the engine failed to load until rails was added as a dependency. Is there a smaller dependency I can use to get the engine to work?